samples/properties_user_links_list.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Analytics Admin API sample application which prints user links under
+the specified parent Google Analytics 4 property.
+
+See https://developers.google.com/analytics/devguides/config/admin/v1/rest/v1alpha/properties.userLinks/list
+for more information.
+"""
+# [START analyticsadmin_properties_user_links_list]
+from google.analytics.admin import AnalyticsAdminServiceClient
+
+from properties_user_links_get import print_user_link
```

Comment:
How are these samples going to be presented in the documentation? Cloud code snippets normally don't import functions from other files since they are intended to be copy-paste runnable.

Reply:
This is a good question. I updated samples to just use print() to output the raw response proto, which does the job just fine, and removes dependencies between samples. Samples for *get() operations (e.g. properties_user_links_get) still have a proper print method that demonstrates how to access and print individual fields of the response.